src/modules/finance/bitcoin.ts

```diff
@@ -0,0 +1,46 @@
+import type { Casing } from '../string';
+
+export enum BitcoinAddressType {
+  Legacy,
+  Segwit,
+  Bech32,
+  Taproot,
+}
+
+export enum BitcoinNetwork {
+  Mainnet,
+  Testnet,
+}
+
+export const BitcoinAddressSpecs = {
```

Comment:
Could we explicitly define the type here? That way we can get rid of the `as Casing` type assertions. Additionally the value interface of each property is described more detailed.

```suggestion
type BitcoinAddressOptions = {
  prefix: Record<BitcoinNetwork, string>;
  length: { min: number; max: number };
  casing: Casing;
  exclude: string;
};

export const BitcoinAddressSpecs: Record<BitcoinAddressType, BitcoinAddressOptions> = {
```